kustomization.yaml

```diff
@@ -0,0 +1,24 @@
+# Adds namespace to all resources.
+namespace: kube-system
+
+# Value of this field is prepended to the
```

Comment:
If we are not using any of these keys then why not delete them all? Without them I don't have to guess whether I need them or not.

Reply:
Removed